### DISCUSSION TITLE
what is best and most popular UI component library for Tailwind CSS

### DISCUSSION THREAD

**Discussion by Eng1Mahmoud** [tailwindlabs/tailwindcss#15061] [General]
what is best and most popular UI component library for Tailwind CSS

**Comment by sanbei101** [comment]
daisyui

**Reply by Eng1Mahmoud** [reply]
i alredy use it,  but i look of if other @sanbei101

**Comment by zoltanszogyenyi** [comment]
Flowbite (https://github.com/themesberg/flowbite) one of them for sure :)

**Comment by grmithu** [comment]
Flowbite

**Comment by abhidave001** [comment]
@Eng1Mahmoud 

There are some good libraries like, Daisy, Meraki, Mamba which are established libraries.
In case you want to try a new one, I suggest FlyonUI [Tailwind Component](https://flyonui.com/) library. 

It is gradually getting popular among the developers.

**Comment by rashadmehtab** [comment]
Take a look at Frutjam (https://frutjam.com/). It works with any framework, scales easily, and is built with an accessibility-first mindset (great color contrast out of the box). Plus, it has zero heavy JS bundles so your PageSpeed results will be incredible.

Source code is here: https://github.com/nezanuha/frutjam


If you are using frameworks like Svelte, Next.js, or Nuxt, they are designed to optimize performance and SEO. However, using heavy JavaScript UI libraries, such as standard modals, often compromises those benefits.

Frutjam solves this problem. As a native-first UI library with zero JS overhead, it works with any framework or project. It preserves your setup's speed and SEO advantages while remaining fully customizable and easy to integrate.
